src/main/java/InterfaceAdaptors/CreateStudentInterfaceAdaptors/CreateStudentPresenter.java

```diff
@@ -0,0 +1,13 @@
+package InterfaceAdaptors.CreateStudentInterfaceAdaptors;
+import CreateStudentUsecase.CreateStudentOutputBoundary;
+import CreateStudentUsecase.CreateStudentResponseModel;
+
+public class CreateStudentPresenter implements CreateStudentOutputBoundary{
+
+    @Override
+    public String getSuccessMessage(CreateStudentResponseModel responseModel) {
+        return "You have successfully created a New student! Welcome" + " " + responseModel.getUsername();}
+
+    @Override
+    public String getFailMessage() {
+        return "Sorry it appears that either the Username already exists or you provided two different usernames";}}
```

Comment:
In the future, instead of returning a String, we should instead raise an Error (one that we create like a CreateStudentFail) that another class can catch. But for now this works well !